### DISCUSSION TITLE
Download the data partially

### DISCUSSION THREAD

**Discussion by amrirasyidi** [torchgeo/torchgeo#1975] [Q&A]
Is there a way to use only fractions of the data?
As an example, the total size of training data from LEVIR+CD is ~3.5GB, let's say I only want to download 3-4 images and masks

**Comment by adamjstewart** [comment]
This is possible for datasets on Source Cooperative (formerly Radiant MLHub) which aren't compressed, but isn't possible in general for most datasets.
